lua/bufferline/render.lua

```diff
@@ -955,6 +955,7 @@ function render.update(update_names, refocus)
     return
   elseif result ~= last_tabline then
     set_tabline(result)
+    vim.api.nvim_exec('redrawt', false)
```

Comment:
The imported `command('redrawtabline')` (alias of `nvim_command`) should suffice here, since it's only one line of VimL (whereas `nvim_exec` is good for multiple lines, but is slower)